### PR TITLE
Optimized the log creation

### PR DESCRIPTION
**Pull request by 244372610**
## Description

Optimized the log creation

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist:

- [ ] Code compiles correctly.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: internal/dflog/loginit.go [hunk 1]
```diff
@@ -179,5 +173,48 @@ func InitDfget(console bool, dir string) error {
...
+		sugar := log.Sugar()
+		SetCoreLogger(sugar)
+		SetGrpcLogger(sugar)
+		SetGCLogger(sugar)
+		SetStorageGCLogger(sugar)
+		SetKeepAliveLogger(sugar)
+		SetStatSeedLogger(log)
+		SetDownloadLogger(log)
+		SetJobLogger(sugar)
+	}
```
Comment (jim3ma) [inline internal/dflog/loginit.go:196]: 这里也重置一下 `levels = nil`吧，init 里面有创建过，重置一下，释放掉多余的资源

**Review by jim3ma — CHANGES_REQUESTED** [review]

File: internal/dflog/loginit.go (hunk 1 above)
Comment (244372610) [reply internal/dflog/loginit.go:197]: done

**Review by hyy0322 — APPROVED** [review]

**Review by jim3ma — APPROVED** [review]
LGTM

**Review by qwtsc — APPROVED** [review]
lgtm

**Pull request merged**